snapcraft/internal/project_loader/grammar/_processor.py

```diff
@@ -19,7 +19,7 @@
 from .errors import GrammarSyntaxError
 
 _ON_CLAUSE_PATTERN = re.compile(r'\Aon\s+')
-_TO_CLAUSE_PATTERN = re.compile(r'\Ato\s+')
+_TO_CLAUSE_PATTERN = re.compile(r'(\Aon\s+\S+\s+|)to\s+')
```

Comment:
Does this not preclude its use standalone? It seems to break items 4, 5, and 6 of [this list](https://forum.snapcraft.io/t/specifying-architecture-on-cross-compilations/1641/20?u=kyrofa).